_episodes_rmd/60-next-steps.Rmd

```diff
@@ -181,7 +181,7 @@ dim(rowData(se))
 ### Subsetting a SummarizedExperiment
 
 SummarizedExperiment can be subset just like with data frames,
-with numerics or with characters of logicals.
+with numerics or with characters of logic.
```

Comment:
this should be logicals, as it refers to the R data type

Reply:
done 🔨